src/Addons/34thPRC_AceArsenalExtended/data/aax_models/vehicles/armour_standard/mkvb/shoulders/config.hpp

```diff
@@ -0,0 +1,65 @@
+class 34thPRC_Mjolnir_Shoulders
+{
+	label = "[34th] Mjolnir MKV[B] Shoulders";
+	author = "34th PRC Modding Team, Vasya";
+	options[] = {"shoulders"};
+	class shoulders
+	{
+		label = "Shoulder Type";
+		alwaysSelectable = 1;
+		changeingame = 0;
+		values[] = {"none", "commando", "cqc", "eva", "grenadier", "gungnir", "jfo", "mkv", "odst", "operator", "recon", "scout", "security"};
+		class none
+		{
+			label = "None";
```

Comment:
I would have used `invisible` but I can see that this has already been merged. We can leave it as is or change it later.